model/pull_request.go

```diff
@@ -29,5 +29,6 @@ type PullRequest struct {
 	URL                 string
 	CreatedAt           time.Time
 	Merged              sql.NullBool
+	MergeCommitSHA      string `db:"-"`
```

Comment:
Did you check what happen with the current SQL query if the CommitSHA is null?

Reply:
I dont understand, this field is not added to the DB. 

The cherry-pick code is called either when a PR is closed or the label is added. Both times the PR object is taken from the request, not from the DB. 

Before we do a cherry-pick we check if `MergeCommitSHA` is not empty. If its empty we return error.

Reply:
I’m referring to just double check if the query tries to scan that field and fails. Vanilla gorp should avoid that field and our branch will not. It was just to be sure before deploy the change

Reply:
@ethervoid I have a mattermod instance running locally and the DB accesses look good. I haven't seen an error from grop for the skipped field